src/extension/common/protocol.re

```diff
@@ -30,58 +27,50 @@ module Refmt = {
     inType: codeType,
     outLang: language
   };
-  type payloadSerialized = {
-    outText: string,
-    inLang: string,
-    outLang: string
-  };
+  [@bs.deriving jsConverter]
   type payload = {
     outText: string,
     inLang: language,
     outLang: language
   };
-  type response = result(payload, string);
+  type response = Result.t(payload, string);
```

Comment:
When I compile this code I get an error here. It looks like it should be `Result.result(string, payload);`

I'm guessing it is a version issue for bs-result?